_review/qec-publiccloud.toml

```
@@ -47,4 +49,62 @@ Params = { flavor = "GCE-Updates", build = "%yesterday%-1" }
 [[Groups]]
 Name = "GCE-BYOS-Updates"
 Params = { flavor = "GCE-BYOS-Updates", build = "%yesterday%-1" }
+[[Groups]]
+Name = "GCE-CHOST-BYOS-Updates"
+Params = { flavor = "GCE-CHOST-BYOS-Updates", build = "%yesterday%-1" }
+
+[[Groups]]
+Name = "Server-DVD-Updates"
+Params = { flavor ="Server-DVD-Updates", groupid = "427", build = "" }
+MaxLifetime = 86400
+
+# Incidents
 
+[[Groups]]
+Name = "Azure-BYOS-Incidents"
+Params = { flavor = "Azure-BYOS-Incidents", build = "%yesterday%-1" }
```

```suggestion
Params = { flavor = "Azure-BYOS-Incidents", build = "" }
```

incidents not applying to this pattern